src/components/InventoryGroups/Modals/AddHostToGroupModal.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Modal from './Modal';
+import { addHostToGroup } from '../utils/api';
+import apiWithToast from '../utils/apiWithToast';
+import { useDispatch } from 'react-redux';
+import { CreateGroupButton } from '../SmallComponents/CreateGroupButton';
+import SearchInput from './SearchInput';
+import { fetchGroups } from '../../../store/inventory-actions';
+import { addHostSchema } from './ModalSchemas/schemes';
+import CreateGroupModal from './CreateGroupModal';
+
+const AddHostToGroupModal = ({
+    isModalOpen,
+    setIsModalOpen,
+    modalState,
+    reloadData
+}) => {
+    const dispatch = useDispatch();
+    //we have to fetch groups to make them available in state
+    useEffect(() => {
+        dispatch(fetchGroups());
+
+    }, []);
+    const [isCreateGroupModalOpen, setIsCreateGroupModalOpen] = useState(false);
+
+    const handleAddDevices = (values) => {
+        const { group } = values;
+        const statusMessages = {
+            onSuccess: {
+                title: 'Success',
+                description: `System(s) have been added to ${group.toString()} successfully`
+            },
+            onError: { title: 'Error', description: `Failed to add ${modalState.name} to ${modalState.groupName}` }
+        };
+
+        apiWithToast(
+            dispatch,
+            () => addHostToGroup(parseInt(group.groupId), modalState.id),
```

Comment:
not sure about using `parseInt` here: group id can contain a-z,A-Z letters, can we just pass it without parsing to int?

Reply:
Yes, you are correct